subsys/bluetooth/common/Makefile

```diff
@@ -0,0 +1,3 @@
+ccflags-y +=-I$(srctree)/subsys/bluetooth
+obj-$(CONFIG_BLUETOOTH_DEBUG) += log.o
```

Comment:
why not debug.c ? The content here are conditionally compiled with CONFIG_BLUETOOTH_DEBUG.